DXRando/DeusEx/Classes/FlagToggleTrigger.uc

```diff
@@ -0,0 +1,37 @@
+class FlagToggleTrigger extends Trigger;
+
+var name flagName;
+var int flagExpiration;
+
+static function FlagToggleTrigger CreateFTT(Actor a, name event, name flagName, optional int flagExpiration)
+{
+    local FlagToggleTrigger ftt;
+
+    ftt = a.Spawn(class'FlagToggleTrigger',, event);
+
+    ftt.flagName = flagName;
+    if (flagExpiration == 0)
+        ftt.flagExpiration = -1;
```

Comment:
-1 doesn't actually work, iirc (Die would know for sure).  I think we fix these by setting them to 999?  Or maybe that's just a FlagTrigger bug?